10up/simple-google-news-sitemap
# Address code review feedback

### Description of the Change

This addresses all but one of the items in issue #12. Open to any feedback or pushback on these items, especially since I'm the one that flagged these items and then fixed them :)

Closes #12 

### How to test the Change

This isn't a major change but a bunch of smaller changes touching most files. As such, just a general test to ensure the plugin still functions as expected is the best test strategy.

### Changelog Entry

> Added - Made sure all custom hooks have docblocks
> Changed - Updated readmes to change structure a bit and add additional information
> Changed - Ensured our minimum PHP version was set the same across all files
> Changed - Modify how we determine which post types to support by default
> Changed - Move hooks from a constructor to an init method
> Changed - Renamed our utility class to `CacheUtils` as it currently only handles caching functionality
> Changed - Add a filter around the post statuses that we clear the cache on

### Credits

Props @dkotter


### Checklist:
